decidim-admin/app/commands/decidim/admin/update_component_permissions.rb

```diff
@@ -1,26 +1,28 @@
 # frozen_string_literal: true
 
+require "hashdiff"
```

Comment:
We should move this to the `decidim-admin/lib/decidim/admin/engine.rb` file, I think